File: src/taburet/accounting/_design/accounting/views/exact_balance_for_account/map.js
function(doc) {
	if (doc.doc_type && doc.doc_type == 'Transaction') {
		emit([doc.from_acc[doc.from_acc.length-1], doc.date[0], doc.date[1], doc.date[2]], {kredit:doc.amount, debet:0})
		emit([doc.to_acc[doc.to_acc.length-1], doc.date[0], doc.date[1], doc.date[2]], {kredit:0, debet:doc.amount})
	}
}

File: src/taburet/transactions/_design/transactions/views/exact_balance_for_account/map.erl
fun({Doc}) ->
	case proplists:get_value(<<"doc_type">>, Doc) of
		<<"Transaction">> ->
			Amount = proplists:get_value(<<"amount">>, Doc),
			[Year, Month, Day|_] = proplists:get_value(<<"date">>, Doc), 
			FromAccId = lists:last(proplists:get_value(<<"from_acc">>, Doc)),
			ToAccId = lists:last(proplists:get_value(<<"to_acc">>, Doc)),
			
			Emit([FromAccId, Year, Month, Day], {[{debet, 0.0}, {kredit, Amount}]}),
			Emit([ToAccId, Year, Month, Day], {[{debet, Amount}, {kredit, 0.0}]});
		_ -> ok
	end
end.
